infrastructure/cdn-in-a-box/docker-compose.yml

```diff
@@ -87,6 +89,7 @@ services:
     depends_on:
       - db
       - enroller
+      - trafficvault
```

Comment:
this won't fix the problem of trafficvault not being available for trafficops use -- it does not enforce startup order,  only build dependencies.

That conflicts with one of the goals we had:  ability to build and test trafficops in isolation.

Reply:
Sorry that I didn't know it need the ability to build and test trafficops in isolation
I can remove it to meet this requirement.

But on the other hand, It does guarantee startup order of container.
Do you mean that this is not enough?
Should I add more steps in `set-to-ips-from-dns.sh` to guarantee that trafficvault registered its DNS record before `set-to-ips-from-dns.sh` add trafficvault IP to `trafficcontrol/infrastructure/cdn-in-a-box/traffic_ops_data/servers/040-trafficvault_server.json`?

Reply:
adding TV as a dependency of TO guarantees that it will be _started_ before TO, but that doesn't guarantee it will be _done_ starting by the time TO needs it. "starting" the container is an instantaneous action that spins off the container's `CMD` as an asynchronous procedure. Ordering within that cannot be guaranteed. I think your idea about using the enroller could work, if I understand you right.

Reply:
@ocket8888 You are right. that what I want to say.
@dangogh Since it need more work, I'll remove this line first and then send it in another PR.

Reply:
Please refer to #3287 for this dependency problem